src/test/java/com/google/api/codegen/testdata/discoveries/go/go_logging.v2beta1.json.baseline

```diff
@@ -22,25 +25,36 @@ import (
 
 func main() {
   ctx := context.Background()
+
+  // Authentication is provided by the `gcloud` tool when running locally
+  // and by built-in service accounts when running on GAE, GCE, or GKE.
+  // For more information, see
+  // https://developers.google.com/identity/protocols/application-default-credentials
   hc, err := google.DefaultClient(ctx, logging.CloudPlatformScope)
   if err != nil {
     // TODO: Handle error.
+    log.Fatal(err)
   }
-  c, err := logging.New(hc)
+
+  service, err := logging.New(hc)
```

Comment:
Do you prefer `c`? I'm fine with it. We can also do `loggingService` - whichever makes more sense.